packages/vars/src/colors.css

```diff
@@ -71,6 +71,11 @@
   --color-green-800: #0c381f;
   --color-green-900: #061c0f;
 
+  /* -- fantasy name of "whatsapp" color to be defined -- */
+  --color-whatsapp-500: #59ce72;
+  --color-whatsapp-600: #33a84c;
+  --color-whatsapp-700: #148a2d;
+
```

Comment:
it is really strange, why this green color is not in the color palette of the default theme. Could you please clarify this thing? It should be looks like extends the colors success palette by adding extra colors

Reply:
@derek-heycar https://www.figma.com/file/rrvkvQEoVTOHa7MzyKSaoz/OTP-Global-Design-System?node-id=1291%3A35195 I checked these colors from global design color pallette, they called as whatsapp

Reply:
@esraltintas ah, yes, I found it